docs/advanced/featureflags.md

```diff
@@ -157,15 +157,20 @@ _See also: [Resource Packs](../resources/index.md#assets), [Data Packs](../resou
 
 Feature packs are a type of pack that not only loads resources and/or data, but also has the ability to toggle on a given set of feature flags. These flags are defined in the `pack.mcmeta` JSON file at the root of this pack, which follows the below format:
 
+:::note
+This file differs from the one in your mod's `resources/` directory. This file defines a brand new feature pack and as thus must be in its own folder.
```

Comment:
```suggestion
This file differs from the one in your mod's `resources/` directory. This file defines a brand new feature pack and 
thus must be in its own folder.
```

Reply:
Why add the new line here? this goes against the rest of the file and overall format from what ive seen, where everything is on 1 line.
Also feels weird to me to add new line midway through a sentence.

Reply:
No idea what GitHub did there, the intention was just to delete a misplaced word. Fixed the suggestion.